.github/ISSUE_TEMPLATE/refactor_tickets.md

```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the bug is.-->
+
+### Motive
+<!-- What is the purpose of this refactoring? If it's removing depcrecated code, please link to the deprecation notice. -->
+
```

Comment:
Would a section `Related issues` be helpful, to point out where this came up? Or is it too much? I usually add it under `Additional context` when its relevant but know that not everyone does. :smile:

I'm ok with it either way.